### DISCUSSION TITLE
Does the webview have to use https to display properly?

### DISCUSSION THREAD

**Discussion by MrZmcH** [coder/code-server#5808] [Q&A]
If I don't want to use https, what can I do to make the webview work?

**Comment by jsjoeio** [comment]
Hmm...you'd have to access in a secure environment. I think localhost is considered secure so that could work. For instance, I run code-server on 8080 on my mac laptop and access it on that device directly.

**Reply by Pythoneryeah** [reply]
How is remote access implemented over http？

**Reply by code-asher** [reply]
The same as HTTPS (web sockets), but browsers refuse to allow certain features over insecure contexts like service workers.
